l1-gateway/README.md

```diff
@@ -8,4 +9,41 @@ target for testing the entire framework end-to-end.
 It may also prove useful for contracts that wish to trustlessly establish the content of storage variables of other contracts,
 or historic values for storage variables of any contract.
 
-For a detailed readme and usage instructions, see the [monorepo readme](https://github.com/ensdomains/evmgateway/tree/main).
+## How to use l1-gateway locally via cloudflare dev env (aka wrangler)
+
+```
+cd l1-gateway
+yarn
+touch .dev.vars
+## set WORKER_PROVIDER_URL
+yarn dev
+```
+
+## How to deploy l1-gateway to cloudflare
+
+```
+cd l1-gateway
+npm install -g wrangler
+wrngler login
+wrangler secret put WORKER_PROVIDER_URL
+yarn deploy
+```
+
+## How to test
+
+```
+cd ../l1-verifier l1-gateway
+PROVIDER_URL=$PROVIDER_URL TARGET_ADDRESS=$TARGET_ADDRESS yarn remote_test
+```
+
+## Current deployments
+
+## L1 Gateway
+
+- Verifier = [0xA63381212fBf7AA2D0b4a2cAe453f8c361B11d06](https://goerli.etherscan.io/address/0xA63381212fBf7AA2D0b4a2cAe453f8c361B11d06)
```

Comment:
Should probably title this section with 'Goerli'. And I don't think addresses for TestL1 and TestL2 are needed in the docs.

Reply:
I will comment as Example, as we also have an instruction of how to test that requires to know l1 test address